src/os/layer/tile.js

```diff
@@ -326,6 +362,46 @@ os.layer.Tile.prototype.setColor = function(value, opt_options) {
   }
 };
 
+/**
+ * Set the brightness for the tile layer.
+ * @param {?number} value The new color
+ * @param {Object=} opt_options The layer options to use
+ */
+os.layer.Tile.prototype.changeBrightness = function(value, opt_options) {
```

Comment:
The set calls for brightness/contrast/saturation are defined by OpenSphere in `layerbasemixin.js` and are only used for tile layers. We put them in that mixin after OpenLayers removed the functions from their base layer class, but we never used them for anything but tile layers. It would be a nice code cleanup pass to:

1. Remove the mixin (get and set calls).
2. Use `set*` instead of `change*` here for consistency.
3. Move the related functions in `os.ui.layer.DefaultLayerUICtrl` to `os.ui.layer.TileLayerUICtrl`.

Thoughts on this @wallw-bits?

Reply:
I agree overall, but what about raster layers that don't use tiles? KML Ground Overlays come to mind (`os.layer.Image`).

Reply:
I guess the base mixin can stay so all layers have those functions, and tile/image layers would have to override them for their specific implementation.